api/Navigator.json

```diff
@@ -4303,7 +4303,7 @@
               "version_added": "8.0"
             },
             "webview_android": {
-              "version_added": false
+              "version_added": "63"
```

Comment:
Although I don't think WebView can be controlled by WebDriver, the `navigator.webdriver` property should exist and return false, so this seems right.